graph-builder/src/lib.rs

```diff
@@ -36,3 +36,8 @@ pub mod graph;
 pub mod registry;
 pub mod release;
 pub mod status;
+
+/// Build info
+pub mod built_info {
```

Comment:
are you planning to use this from somewhere else? Otherwise it doesn't need to be public.

Reply:
It might even be possible to put this into the `commons` crate and use it from the GB and PE. Have you tried that?

Reply:
Removed `pub` (also fixed warnings during build as some const are not being used).

>It might even be possible to put this into the commons crate and use it from the GB and PE. Have you tried that?

I'm not sure if that would work. GB and PE could be built and run independently, not sure what would build_info info from a common crate would show. I'd rather keep these separate for now - and attempt to merge it later

Reply:
I'm fine postponing this